src/CloudIoTCoreMqtt.cpp

```diff
@@ -193,10 +193,14 @@ bool CloudIoTCoreMqtt::publishTelemetry(String subtopic, const char* data, int l
 }
 
 // Helper that just sends default sensor
-bool CloudIoTCoreMqtt::publishState(String data) {
+bool CloudIoTCoreMqtt::publishState(String &data) {
```

Comment:
Consider using `const String &data` - I assume this does not affect calling clients?

Reply:
you're right - updated